DigitalLearningSolutions.Data.Tests/Services/TutorialContentServiceTests.cs

```diff
@@ -241,20 +252,53 @@ public void Get_tutorial_information_nextSection_skips_sections_full_of_archived
         [Test]
         public void Get_tutorial_information_nextSection_returns_section_with_only_diagnostic_assessment()
         {
-            // Given
-            const int candidateId = 74411;
-            const int customisationId = 5852;
-            const int sectionId = 150;
-            const int tutorialId = 634;
+            using (new TransactionScope())
+            {
 
-            const int expectedNextSectionId = 151; // All tutorials are CustomisationTutorials.Status = 0, though some DiagStatus = 1
+                // Given
+                const int candidateId = 74411;
+                const int customisationId = 5852;
+                const int sectionId = 150;
+                const int tutorialId = 634;
 
-            // When
-            var tutorial = tutorialContentService.GetTutorialInformation(candidateId, customisationId, sectionId, tutorialId);
+                const int expectedNextSectionId = 151; // All tutorials are CustomisationTutorials.Status = 0, though some DiagStatus = 1
+                // Remove post learning assessment
+                tutorialContentTestHelper.UpdatePostLearningAssessmentPath(expectedNextSectionId, null);
 
-            // Then
-            tutorial.Should().NotBeNull();
-            tutorial!.NextSectionId.Should().Be(expectedNextSectionId);
+                // When
+                var tutorial = tutorialContentService.GetTutorialInformation(candidateId, customisationId, sectionId, tutorialId);
+
+                // Then
+                tutorial.Should().NotBeNull();
+                tutorial!.NextSectionId.Should().Be(expectedNextSectionId);
+            }
+        }
+
+        [Test]
+        public void Get_tutorial_information_nextSection_skips_section_with_no_diagnostic_assessment_path()

```

Comment:
Is this a next section with just a diagnostic assessment but no diagnostic assessment path? If so then I think it's worth making that clear in the name, at the moment it sounds like it will skip any section without a diagnostic assessment path 🙂

Reply:
Yep! Changed in 3f3f8da 🙂